### DISCUSSION TITLE
Organize import custom group ordering

### DISCUSSION THREAD

**Discussion by iccowan** [mfussenegger/nvim-jdtls#372] [Q&A] [answered]
I am trying to figure out how to apply custom group ordering when organizing the imports. My project has a `checkstyle.xml` file in it, but I cannot find a way to tell the JDTLS about this file or to manually configure the import groups. Additionally, I cannot figure out if this is a nvim-jdtls issue or if it has something to do with my jdtls configuration. Any help would be appreciated - thanks!

**Comment by abenz1267** [comment, marked as answer]
Import order can be configured like this:

```
 completion = { importOrder = { "java", "javax", "org", "com" } },
```

**Reply by iccowan** [reply]
> Import order can be configured like this:
> 
> 
> 
> ```
> 
>  completion = { importOrder = { "java", "javax", "org", "com" } },
> 
> ```

How do you specify static imports? And is there a way to specify grouping?

**Reply by peri4n** [reply]
Please have a look at `favoriteStaticMembers` on:
https://github.com/eclipse/eclipse.jdt.ls/wiki/Running-the-JAVA-LS-server-from-the-command-line#initialize-request

**Reply by iccowan** [reply]
> Please have a look at `favoriteStaticMembers` on:
> https://github.com/eclipse/eclipse.jdt.ls/wiki/Running-the-JAVA-LS-server-from-the-command-line#initialize-request

Perfect this did it for static imports. Thanks!
